src/licensedcode/data/licenses/ekioh.yml

```diff
@@ -4,7 +4,6 @@ short_name: Ekioh License
 name: Ekioh License
 category: Permissive
 owner: Ekioh
-notes: see also https://kryogenix.org/code/browser/licence.html
```

Comment:
May be this can be added to the bottom on the remaining notes field?

Reply:
Merged.